src/components/SignInButton.tsx

```diff
@@ -1,22 +1,20 @@
 "use client";
 
-import { useSearchParams } from "next/navigation";
 import { signIn } from "next-auth/react";
 import { ReactNode } from "react";
 
-type PropsType = {
-  children: ReactNode
-  provider: string
-}
+type LoginProvidersType = "github"; // | "google"
```

Comment:
IMO, the suffix "Type" is kind of redundant, as we already say it's a type when declaring it

Reply:
I changed it. I thought this was good practice as you can easily see the difference between Types and Interfaces this way (also, this is how my youtube professor used to name them hahaha).

Anyway, commit [2fefceb](https://github.com/podcodar/app/pull/15/commits/2fefcebf36ccdb02ea095c2c13cbe9ead64d8d37) make the changes you asked for.